### DISCUSSION TITLE
please tell me how can I find the pinio1 contact

### DISCUSSION THREAD

**Discussion by Suharevsky171296** [iNavFlight/inav#10508] [General]
please tell me how can I find the pinio1 contact for the speedybee f405v4 flight controller or maybe I can somehow reassign it to another pin?

**Comment by mmosca** [comment]
The target defines it as PB11. Check the manufacturer's documentation for what it is used for.

It is likely not exposed as a pad, but may be turning on/off power to the vtx or switching analog cameras.

**Reply by Suharevsky171296** [reply]
There is no contact for it on the flight controller. In another configurator I simply reassigned pinio and that’s it. But since this option is not available here, I understand that you can only change and compile through the firmware and then flash it

**Reply by mmosca** [reply]
That is not a feature in INAV. You would have to create a custom target and build custom firmware for this.

**Reply by Suharevsky171296** [reply]
thank you I will try
